script/download.js

```diff
@@ -21,16 +21,21 @@ const parseArgs = require('./parseArgs');
 const getConfigHash = require('../src/get-config-hash');
 
 
-const {repo, cdn} = parseArgs();
+const {repo, plat, cdn} = parseArgs();
```

Comment:
A description of the plat parameter needs to be added to the file header comments.